registry.json

```diff
@@ -1,4 +1,19 @@
 [
+   {
+      "ticker":"RSV",
+      "poolId":"pool1r0sxz0hygfc0266lhdclve6j3cwcwlth6dya6uey65yxjnwm7qc",
```

Comment:
Your pool id is : 1be0613ee44270f56b5fbb71f667528e1d877d77d349dd7324d50869
Please update your PR.